superset-frontend/src/components/DvtDragCard/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import Icon from '../Icons/Icon';
+import {
+  StyledDvtCard,
+  StyledDvtCardSize,
+  StyleddvtCardCard,
+  StyleddvtCardIcon,
+  StyleddvtCardLabel,
+} from './dvt-drag-card.module';
+
+export interface DvtDragCardData {
+  label: string;
+  value: string;
+  icon: string;
+}
+
+export interface DvtDragCardProps {
+  data: DvtDragCardData[];
```

Comment:
data: DvtDragCardData[];  array değil tek object kalmalı tek item için

item: DvtDragCardData olabilir